draw/reddit.dart

```diff
@@ -0,0 +1,9 @@
+// Provides the reddit class
+
+class reddit{
```

Comment:
Take a look at this page here for documentation styling: https://www.dartlang.org/guides/language/effective-dart/documentation. If we do it right, dartdoc automatically generates documentation pages when we upload the library to pub!